src/main/resources/bootstrap.yaml

```diff
@@ -78,3 +78,4 @@ spring:
         s2s.microservicekey-pcq-consolidation-service: microserviceKeys.pcq_consolidation_service
         s2s.microservicekey-aac-manage-case-assignment: microserviceKeys.aac_manage_case_assignment
         s2s.microservicekey-ucmc-service: microserviceKeys.ucmc_service
+        s2s.microservicekey-unspec-service: microserviceKeys.unspec_service
```

Comment:
Hi Piotr, I've checked the s2s keyvaults for this and I can only see it in aat and demo. The pull request template details how this should be added to all https://github.com/hmcts/service-auth-provider-app/blob/master/.github/PULL_REQUEST_TEMPLATE.md

Reply:
done + updated PR decription